securedrop/tests/__init__.py

```diff
@@ -2,6 +2,11 @@
 from os.path import abspath, dirname, join, realpath
 import pytest
 import sys
+import os
+
+# WARNING: This variable must be set before any import of the securedrop code/app
+# or most tests will fail
+os.environ["SECUREDROP_ENV"] = "test"
```

Comment:
Putting this here ensures that the `SECUREDROP_ENV` will always be set before any application code is imported.